sunpy/net/dataretriever/sources/stereo.py

```diff
@@ -0,0 +1,731 @@
+"""STEREO Dataretriever sources subclass definitions"""
+
+__author__ = "Ankit Kumar"
+__email__  = "ankitkmr.iitk@gmail.com"
+
+#This module was developed under funding provided by
+#Google Summer of Code 2015
+
+import datetime
+
+from astropy import units as u
+from ..client import GenericClient
+
+from sunpy.util.scraper import Scraper
+from sunpy.time import TimeRange
+
+
+
+__all__ = ['SEPTClient', 'HETClient', 'SITClient', 'PLASTICClient', 'MAGClient', 'LETClient']
+
+
+class SEPTClient(GenericClient):
+    def _get_url_for_timerange(cls,timerange, stereo_spacecraft = 'ahead', duration_of_average = 10 * u.min,
+                               specie = 'element', sensor_pointing = 'asun'):
```

Comment:
please rename to `species`, also in the docstring. Seems that's the way it's used...